packages/core/content-manager/admin/src/pages/ListView/components/BulkActions/PublishAction.tsx

```diff
@@ -240,30 +240,32 @@ const SelectedEntriesTableContent = ({
               )}
             </Table.Cell>
             <Table.Cell>
-              <IconButton
-                tag={Link}
-                to={{
-                  pathname: `${pathname}/${row.documentId}`,
-                  search: row.locale && `?plugins[i18n][locale]=${row.locale}`,
-                }}
-                state={{ from: pathname }}
-                label={formatMessage(
-                  { id: 'app.component.HelperPluginTable.edit', defaultMessage: 'Edit {target}' },
-                  {
-                    target: formatMessage(
-                      {
-                        id: 'content-manager.components.ListViewHelperPluginTable.row-line',
-                        defaultMessage: 'item line {number}',
-                      },
-                      { number: index + 1 }
-                    ),
-                  }
-                )}
-                target="_blank"
-                marginLeft="auto"
-              >
-                <Pencil />
-              </IconButton>
+              <Flex>
+                <IconButton
+                  tag={Link}
+                  to={{
+                    pathname: `${pathname}/${row.documentId}`,
+                    search: row.locale && `?plugins[i18n][locale]=${row.locale}`,
+                  }}
+                  state={{ from: pathname }}
+                  label={formatMessage(
+                    { id: 'app.component.HelperPluginTable.edit', defaultMessage: 'Edit {target}' },
+                    {
+                      target: formatMessage(
+                        {
+                          id: 'content-manager.components.ListViewHelperPluginTable.row-line',
+                          defaultMessage: 'item line {number}',
+                        },
+                        { number: index + 1 }
+                      ),
+                    }
+                  )}
+                  target="_blank"
+                  marginLeft="auto"
```

Comment:
If 'remember me' isn't enabled new tabs open to the log in page, which makes
opening the different locales in new tabs a bit frustrating.

Do we want to open locales in _blank ?

Reply:
If I remember well this is intended, the idea is users open the BulkPublish modal and then they can open multiple tabs to edit entries, if we don't use _blank you lose the state of the bulkPublish modal when you go to the other page

So we decided to do it like this even if users without "remember me" are going to have problems